docs/docs_source/scenarios/business.rst

```diff
@@ -1,9 +1,39 @@
 
-..  _Editing business documents:
+..  _Business editing:
 
-|stub-icon| Editing business documents
-======================================
+Supporting document editing for business users
```

Comment:
Do we want to specify that this flow is for business/paid users. We currently call it support for business users but really it's support for paid users. Just an idea.